build.gradle

```diff
@@ -25,7 +25,7 @@ ext {
 apply plugin: 'application'
 apply from: 'gradle/formatting.gradle'
 
-mainClassName = 'opensearchSDK.ExtensionsRunner'
+mainClassName = 'org.opensearch.sdk.ExtensionsRunner'
```

Comment:
Consider namespacing further if that makes sense: `org.opensearch.sdk.extensions#Runner`.